cmd/minikube/cmd/config/validations.go

```diff
@@ -59,6 +58,35 @@ func IsValidURL(name string, location string) error {
 	return nil
 }
 
+func IsURLExists(name string, location string) error {
+	parsed, err := url.Parse(location)
+	if err != nil {
+		return fmt.Errorf("%s is not a valid URL", location)
+	}
+
+	// we can only validate if local files exist, not other urls
+	if parsed.Scheme != "file" {
+		return nil
+	}
+
+	// chop off "file://" from the location, giving us the real system path
+	sysPath := string([]rune(location[len("file://"):]))
+	stat, err := os.Stat(sysPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("%s does not exist", location)
+		}
+
+		return err
+	}
+
```

Comment:
Just to clarify if there are future improvements to make: does this check mean that minikube will ignore the value if the file exists but is not readable?

Reply:
This line that you commented on returns any other sort of error you get when doing "stat" on the file.

I will add a case to handle "no permission" issues.

When stat succeeds, and the path is not a directory, then no error will be returned. Is there any situation where this will happen but the file still cannot be read? I can add another check if you have anything in mind.